Mathlib/LinearAlgebra/Dimension/StrongRankCondition.lean

```diff
@@ -542,3 +542,19 @@ theorem mem_span_set_iff_exists_finsupp_le_finrank :
     exact mem_span_set.mpr ⟨c, hcs, hx⟩
 
 end Submodule
+
+section IsQuadraticAlgebra
+
+/--
+An extension of rings `R ⊆ S` is quadratic if `S` is a free `R`-module of rank `2`.
+-/
```

Comment:
Can you also add this doc at the beginning of the file?

Reply:
No, I didn't add a docstring for the new class at the top of the file in the previous PR so it's normal you can't find it. 

I added one in this PR though.

Reply:
🐙